ansible/roles/logstash-openshift/README.md

```diff
@@ -0,0 +1,17 @@
+# Role to provision logstash
+
+The role isn't idempotent at the moment.
+
+## Provisioning
+
+Run the role to provision `logstash`.
+
+## Deletion
+
+Run the role, pass the variable `logstash_deploy=false` to the role to get
+logstash wiped completely.
+
+## Add your configuration to logstash
+
+Add your configuration files to [{{ role_path }}/files/conf.d](files/conf.d).
```

Comment:
at the moment it requires configuration files before deploy.

updating configs: remove/redeploy.

updating configs w/o redeploy I think would be next step (via kubernetes means).
`reload` - needs to be checked.. I'm not sure how this can be done.